### ISSUE TITLE
Syntax errors with paths

### ISSUE THREAD

**Issue by JakeHedman** [bijoutrouvaille/fireward#12]
Functions returning a path causes ``` function `somePath` is missing a closing `}` ```:
```
function somePath() {
  return /foo/bar
}
```

Using dot notation inside `$()` in paths causes ```missing closing paren on $(```:
```
function userExists() {
  return exists(/users/$(request.auth.uid))
}
```

Would it be useful to add a few large rules files to the test suite as a "smoke test" just to make sure fireward doesn't crash on weird but valid rules? I'm currently using a hacky jest script locally.

**Comment by bijoutrouvaille** [comment]
Thank you for the report @JakeHedman, and apologies for a late reply. I think the smoke test is a good idea. I'll try to get to it within the next couple of days.

**Comment by bijoutrouvaille** [comment]
Fixed the second case (v 1.1.3). The first case needs a little more research. The documentation is not very clear as to when the path is to be parenthesized.

This will work:
```
function somePath() {
  return (/foo/bar)
}
```

**Comment by bijoutrouvaille** [comment]
`make smoke-test` will try to compile `examples/smoke-test.ward`
